src/tag/Type.h

```diff
@@ -46,6 +46,7 @@ enum TagType
 	TAG_NAME,
 	TAG_GENRE,
 	TAG_DATE,
+	TAG_ORIGINAL_DATE,
```

Comment:
I don't know the answer; I wasn't sure where it would be needed and where it would not, and was trying to err on the side of caution. C++ is not my strong suit.